From: SeongJae Park <sj@kernel.org>
Date: Sat, 7 Dec 2024 12:15:40 -0800
Subject: [PATCH] ==== fine-grained damos filtr stat action ====

- Add DAMOS action that same to stat but applies filter
- Expose per-region filtered bytes to API/sysfs users

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 damon_meta_changes/D7jYt0Oq | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 damon_meta_changes/D7jYt0Oq

diff --git a/damon_meta_changes/D7jYt0Oq b/damon_meta_changes/D7jYt0Oq
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

